src/bun.js/bindings/JSMockFunction.cpp

```diff
@@ -1362,12 +1362,9 @@ BUN_DEFINE_HOST_FUNCTION(JSMock__jsSetSystemTime, (JSC::JSGlobalObject * globalO
         }
         return JSValue::encode(callFrame->thisValue());
     }
+    // number > 0 is a valid date otherwise it's invalid and we should reset the time (set to -1)
+    globalObject->overridenDateNow = (argument0.isNumber() && argument0.asNumber() > 0) ? argument0.asNumber() : -1;
```

Comment:
```suggestion
    globalObject->overridenDateNow = (argument0.isNumber() && argument0.asNumber() >= 0) ? argument0.asNumber() : -1;
```
Setting time to `1970-01-01T00:00:00.000Z` should work as well